frontend/src/metabase/home/components/EmbedHomepage/tests/common.unit.spec.tsx

```diff
@@ -59,27 +59,11 @@ describe("EmbedHomepage (OSS)", () => {
     ).not.toBeInTheDocument();
   });
 
-  it("should prompt to enable embedding if it wasn't auto enabled", () => {
-    setup({ settings: { "setup-embedding-autoenabled": false } });
-
-    expect(
-      screen.getByText("Enable embedding in the settings"),
-    ).toBeInTheDocument();
-
-    expect(
-      screen.queryByText("Embedding has been automatically enabled for you"),
-    ).not.toBeInTheDocument();
-  });
-
-  it("should not prompt to enable embedding if it was auto enabled", () => {
-    setup({ settings: { "setup-embedding-autoenabled": true } });
-
-    expect(
-      screen.queryByText("Enable embedding in the settings"),
-    ).not.toBeInTheDocument();
+  it("should prompt to enable embedding", () => {
+    setup();
 
     expect(
-      screen.getByText("Embedding has been automatically enabled for you"),
+      screen.getByText("Enable static embedding in the settings"),
```

Comment:
I don't see we have an equivalent test for interactive embedding. I'd expect to see a test with a string `Enable interactive embedding in the settings`